site/src/api/queries/groups.ts

```diff
@@ -18,10 +18,10 @@ export const groups = (organizationId: string) => {
   } satisfies UseQueryOptions<Group[]>;
 };
 
-export const group = (groupId: string) => {
+export const group = (groupName: string) => {
```

Comment:
We should change the route to /organizations/id/groups/name then. Right now, this would break the scoping that exists.

Reply:
It doesn't make sense to move those _quite_ yet. I specifically advised to put that off for now. Definitely addressed before multi-org ships tho!

Reply:
What about anyone who depends on our API? Are we just going to break them?

Reply:
hmm? the backend hasn't been changed at all, the api is unbroken